### PR TITLE
[WIP] runner-faqs-adoc

### PR DESCRIPTION
**Pull request by thomasr8**
Initial draft of Runner FAQs

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/runner-faqs-adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== How do I get customers on to the preview?
+
+See How to run a CircleCI Runner preview
+
+=== Can I try runner out for myself?
```
Comment (teesloane) [inline jekyll/_cci2/runner-faqs-adoc:16]:
```suggestion
=== Can I try Runner out for myself?
```

File: jekyll/_cci2/runner-faqs-adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Internal users are welcome to give runner a try.
```
Comment (teesloane) [inline jekyll/_cci2/runner-faqs-adoc:18]:
```suggestion
Internal users are welcome to give Runner a try.
```

File: jekyll/_cci2/runner-faqs-adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+The install docs can be found here:
+https://circleci-binary-releases.s3.amazonaws.com/circleci-launch-agent/docs/install.html
```
Comment (teesloane) [inline jekyll/_cci2/runner-faqs-adoc:21]: Links with asciidoctor can be formatted as so:

`https://my-link.com[My link description].`

**Comment by glenjamin** [comment]
The pitch from marketing is that it should be lowercase, unless in title-case

**Comment by glenjamin** [comment]
I think this FAQ would be a better starting point than the Guru FAQ: https://github.com/CircleCI-Public/runner-preview-docs/#faqs

However I think it might be too early to start putting content in the main docs repo, as we'll need to keep tracking changes against the preview docs repo for now

File: jekyll/_cci2/runner-faqs-adoc (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-faqs-adoc]: Fixed

File: jekyll/_cci2/runner-faqs-adoc (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-faqs-adoc]: Fixed

File: jekyll/_cci2/runner-faqs-adoc [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== How do I get customers on to the preview?
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-faqs-adoc:12]: Seems odd to refer to customers? Is this and the item below for internal use only?

File: jekyll/_cci2/runner-faqs-adoc [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+:toc: macro
+:toc-title:
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== What is the pricing for the CircleCI runner?
+
+Currently, CircleCI does not charge for runner usage; however, this will change in the future. If you are a "preview" customer, your account team will keep you informed of any pricing changes for the runner.
```
Comment (appplemac) [inline jekyll/_cci2/runner-faqs-adoc:14]: This is no longer true. We should remove the pricing section for now, we can re-add it and point to the Scale plan once the relevant marketing page is up.

File: jekyll/_cci2/runner-faqs-adoc [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+In order to connect back to CircleCI to receive and execute jobs, outbound HTTPS connections to `runner.circleci.com`, `circleci-binary-releases.s3.amazonaws.com` are required.
+
+NOTE: No inbound connectivity is required by runner. Any other required connectivity is dependent on the content of the jobs themselves.
+
+CAUTION: Using the checkout step will require access to your VCS provider. Using the cache, workspace or artifact features will require outbound HTTPS connections to `circle-production-customer-artifacts.s3.amazonaws.com`.
+
+=== How do caching and workspaces and artifacts work with the runner?
+
+Caches and workspaces and artifacts will be stored in the us-east-1 region of S3. If your runners are not in this region then you may see reduced performance. Although we are not currently charging for runner usage, the transfer and storage of data will incur costs when pricing is introduced.
```
Comment (appplemac) [inline jekyll/_cci2/runner-faqs-adoc:44]: Let’s rephrase the charging part as “we expect to charge for data transfer and storage in the future”.

File: jekyll/_cci2/runner-faqs-adoc (hunk 5 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-faqs-adoc:14]: Removed from document

File: jekyll/_cci2/runner-faqs-adoc (hunk 6 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-faqs-adoc:44]: Fixed

File: jekyll/_cci2/runner-faqs-adoc (hunk 4 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-faqs-adoc]: fixed

File: jekyll/_cci2/runner-faqs-adoc (hunk 3 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-faqs-adoc]: Fixed

**Review by teesloane — CHANGES_REQUESTED** [review]
The file `runner-faqs-adoc` needs to be changed to `runner-faqs`. 

image

**Pull request merged**